backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/views/PP_Maturing_Candidate_v.sql

```diff
@@ -25,14 +28,20 @@ FROM m_hu hu
          INNER JOIN m_maturing_configuration_line mcl ON hus.m_product_id = mcl.from_product_id AND mcl.isactive = 'Y'
          INNER JOIN m_maturing_configuration mc ON mcl.m_maturing_configuration_id = mc.m_maturing_configuration_id AND mc.isactive = 'Y'
          INNER JOIN pp_product_planning pp ON w.m_warehouse_id = pp.m_warehouse_id AND mcl.m_maturing_configuration_line_id = pp.m_maturing_configuration_line_id AND pp.m_product_id = mcl.matured_product_id AND pp.isactive = 'Y' AND pp.ismatured = 'Y'
+         INNER JOIN pp_product_bomversions bomv ON bomv.pp_product_bomversions_id = pp.pp_product_bomversions_id
+         INNER JOIN pp_product_bom bom ON bomv.pp_product_bomversions_id = bom.pp_product_bomversions_id
+         INNER JOIN pp_product_bomline boml ON boml.pp_product_bom_id = bom.pp_product_bom_id AND boml.m_product_id = hus.m_product_id
```

Comment:
avoid joining more than one row!

Reply:
Moved logic to Java side `org.eevolution.productioncandidate.service.PPOrderCandidateService#getQtyRequired`